gluster-block.spec.in

```diff
@@ -52,6 +52,7 @@ rm -rf ${RPM_BUILD_ROOT}
 %{_sbindir}/gluster-block
 %{_sbindir}/gluster-blockd
 %{_mandir}/man8/gluster-block*.8*
+%dir %{_localstatedir}/log/gluster-block
```

Comment:
looks good to me :-)

But [glusterBlockLogdirCreate()](https://github.com/gluster/gluster-block/blob/master/daemon/gluster-blockd.c#L54 ) will do the needful already right ? wasn't that sufficient ?